### ISSUE TITLE
Installation Error with Node v6.0

### ISSUE THREAD

**Issue by azeemdwt** [gmetais/YellowLabTools#157]
Hi, 

I am getting this error while trying to install YellowLabTools 

OS: Ubuntu 13.10 32 bit
Nodejs Version: 6.0.0
npm  v3.9.0

```
npm ERR! Linux 3.11.0-26-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "yellowlabtools"
npm ERR! node v6.0.0
npm ERR! npm  v3.9.0
npm ERR! code ELIFECYCLE

npm ERR! lwip@0.0.8 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the lwip@0.0.8 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the lwip package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs lwip
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls lwip
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/username/npm-debug.log
npm ERR! code 1`
```

**Comment by gmetais** [comment]
Hi!

This is a problem with Node 6.x. Have a look here: https://github.com/EyalAr/lwip/issues/249

For the moment, the only option is to downgrade Node, sorry. I'll update YLT as soon as the bug is fixed in lwip.

Thanks for reporting the issue!

**Comment by azeemdwt** [comment]
@gmetais : Thanks for your kind response. Much appreciated.

**Comment by gmetais** [comment]
Fixed by #160.

Just update to YLT v1.10.1 and it should work!
